src/components/item/ItemLanguageSelector.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <b-container
+    fluid
+    class="border-bottom mt-3"
+  >
+    <b-container>
+      <b-row>
+        <b-col class="col-12 pb-3 d-inline-flex align-items-center flex-wrap">
+          <span>
+            <span class="icon-language pr-2" />
+            <i18n
+              path="multilingual.currentLanguage"
+              tag="span"
+              class="pr-1"
+              data-qa="translate item suggestion"
+            >
+              <span v-if="!translated && itemLanguage === selectedLocale.code">
+                a different language
+              </span>
+              <b-link
```

Comment:
It probably makes more sense to have this and the links in the dropdown to be Nuxt-links with a :to property that can call a method 'translateParams' where it can return:
```
translateParams(language) {
  const queryParams = {};
  if (language) queryParams.metadataLang = language;
  return { path: this.$route.path, query: queryParams };
}
```
That way 'metadataLang' is removed also when  'resetting' to the edmLanguage.